### ISSUE TITLE
import HybrIK-X pickle file inside blender

### ISSUE THREAD

**Issue by elia-orsini** [Jeff-sjtu/HybrIK#173]
hello,
is it already possible to import HybrIK-X animations in blender via the Blender add-on ? 
is this feature being developed ? 
I could try implementing it and opening a PR otherwise.

**Comment by nbartzokas** [comment]
Yes. Use the `--save-pk` option as shown [here](https://github.com/Jeff-sjtu/HybrIK#smpl) and the system will write a `res.pk` file. You can import that into Blender using the HybrIK add-on by clicking its "Import Pickle Results" and selecting `res.pk`.
